submissions/jeffersonMormah/htmlForm/signup.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
@Jeffbuchi Well done!. You did not link the contact page and product page in this file.